chia/full_node/full_node_api.py

```diff
@@ -141,14 +142,20 @@ async def request_transaction(self, request: full_node_protocol.RequestTransacti
 
     @peer_required
     @api_request
+    @bytes_required
     async def respond_transaction(
-        self, tx: full_node_protocol.RespondTransaction, peer: ws.WSChiaConnection, test: bool = False
+        self,
+        tx: full_node_protocol.RespondTransaction,
+        peer: ws.WSChiaConnection,
+        tx_bytes: bytes = b"",
```

Comment:
you added the `tx_bytes` parameter here, but I don't see any call sites updated to pass this in,

Reply:
Decorator `@api_request` will for each instance of non bytes paramter named 'param_name' inject parameter "param_name_bytes" 
(if `bytes_required` decorator is also present)